CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * This library requires Turf v2.0.0-alpha.3. ([#525](https://github.com/mapbox/mapbox-directions-swift/pull/525))
```

Comment:
should these comments be under v1.2.x or v2.0?

Reply:
It is a little messy right now: we haven’t put out a v1._x_ release but could do so in conjunction with navigation SDK v1.4.0. Once that happens, there will be a v1.4 section below, but this section is actually for v2.0.0 and just hasn’t been updated to say so yet.